README.md

```diff
@@ -25,7 +25,7 @@ You can learn more about developing libraries for Dynamo on the [Dynamo wiki](ht
 ### Build Dynamo from Source ###
 You will need the following to build the latest Dynamo on Windows:
 
-- [Microsoft Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) (any edition)
+- [Microsoft Visual Studio 2019 2019.16.8](https://visualstudio.microsoft.com/downloads/) (any edition)
```

Comment:
Why 2019 2 times?

Reply:
I thought that was the part of the version no.

Reply:
copy and paste error ;)